submissions/AnnaStoyano/js-dom/script.js

```diff
@@ -0,0 +1,105 @@
+const downloadInfoFromJson = function () {
+    let promise = fetch('./information.json');
+    promise.then(res => res.json())
+        .then(data => {
+            addMenuItems(data.faculties);
+            addContentItems(data.faculties);
+            return data;
+        })
+        .then(data=>{
+            menuHandler(data);
+            menuIconHandler();
+        })      
+}
+
+document.addEventListener('DOMContentLoaded',downloadInfoFromJson);
+
+const addMenuItems = function (arr) {
+    let menu = document.querySelector('.menu');
+    let menuFr = document.createDocumentFragment();
+    arr.forEach((element, index) => {
+        let menuItem = document.createElement('li');
+        let link = document.createElement('a');
+        link.setAttribute('href', '#');
+        link.textContent = element.name;
+        menuItem.insertAdjacentElement('afterbegin', link);
+        menuItem.classList.add("menu-item", element.name.toLowerCase());
+        if (index === 0) {
+            menuItem.classList.add('active');
+        }
+        menuFr.appendChild(menuItem);
+    });
+    menu.appendChild(menuFr);
+}
+
+const addContentItems = function (arr) {
+    let content = document.querySelector('.content');
+    let contentFr = document.createDocumentFragment();
+
+    arr.forEach((element, index) => {
+        let contentItem = document.createElement('div');
+        let figure = document.createElement('figure');
+        let img = document.createElement('img');
+        let desc = document.createElement('div');
+        img.setAttribute('src', element.img);
+        img.setAttribute('alt', element.name + '-logo');
+        figure.appendChild(img);
+        desc.textContent = element.description;
+        desc.classList.add('description');
+        figure.classList.add('logo');
+        contentItem.insertAdjacentElement('afterbegin', figure);
+        contentItem.insertAdjacentElement('beforeend', desc);
+        contentItem.classList.add('content-item',element.name.toLowerCase());
+        if (index === 0) {
+            contentItem.classList.add('active');
+        }
+        contentFr.appendChild(contentItem);
+    })
+
+    content.appendChild(contentFr);
+}
+
+const menuIconHandler = function () {
```

Comment:
It's better to start function name from a verb. More info you can find in this [article](https://medium.com/javascript-in-plain-english/javascript-naming-convention-best-practices-b2065694b7d) in a section about the functions